platform/android/src/native_map_view.cpp

```diff
@@ -555,7 +555,7 @@ void NativeMapView::setVisibleCoordinateBounds(JNIEnv& env, jni::Array<jni::Obje
 }
 
 void NativeMapView::setContentPadding(JNIEnv&, double top, double left, double bottom, double right) {
-    insets = {top, left, bottom, right};
+    insets = EdgeInsets(top, left, bottom, right);
```

Comment:
You can use the `{{ }}` notation here has well. (common in the rest of the code base)